src/main/java/frc/robot/pneumatics/Pneumatics.java

```diff
@@ -0,0 +1,77 @@
+package frc.robot.pneumatics;
+
+import edu.wpi.first.wpilibj.Compressor;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.littletonrobotics.junction.Logger;
+
+public class Pneumatics extends SubsystemBase {
+
+  // For this implementation we are going to have two solenoids for Susan's intake //
+  // Look at slides to see Susan intake //
+  private DoubleSolenoid rightSolenoid;
+  private DoubleSolenoid leftSolenoid;
+
+  // We dont use the compressor but it needs to be extantiated anways //
+  // This is to make sure that the compressor holds air for the solenoids //
+  private Compressor compressor;
+
+  public Pneumatics() {
+
+    // Use the PneumaticsConstants class to get the forward and reverse channel of the Double Solenoid //
+    // Also, if you are using a CTRE pneumatics hub change 'REVPH' to 'CTREPM' //
+    // Module number is default going to be '1' //
+    // Read here to learn more about it -> https://docs.wpilib.org/en/stable/docs/software/hardware-apis/pneumatics/index.html //
+    rightSolenoid =
+        new DoubleSolenoid(
+          PnematicsConstants.moduleID,
```

Comment:
Both left and right use the same moduleID?

Reply:
Yes, they both use the same module ID. The module ID isn't like a port on motors but it just specifies what pneumatic hub they feed back to.

Reply:
![image](https://github.com/user-attachments/assets/edf3abc3-9f84-4baf-8710-6cfa292617aa)